keps/sig-release/2572-release-cadence/README.md

```diff
@@ -0,0 +1,389 @@
+
+*Kubernetes Release Sechedule 2022 (Proposed 3 Release Cadence)*
+
+| Year Week Number | Release Number | Release Week | Note |
+| -------- | -------- | -------- | -------- |
+| 1  | 1 | 1 (January 3) | |
+| 15 | 1 | 15 (April 12) | | 
+| 18 | 2 | 1 (May 2) | Probably KubeCon EU |
+| 33 | 2 | 15 (August 15) | |
+| 36 | 3 | 1 (September 6 | Probably KubeCon NA | 
+| 51 | 3 | 15 (December 20) | 
+
+### User Stories
+
+Kubernetes releases are made by real people. The technical aspects—for example,
+the release automation—reflects only a tiny part of the complete cycle. This
+means we will mainly focus on the human aspects and their corresponding roles
+when deciding to move to a 3-releases-per-year cadence.
+
+#### End User
+
+Most end user organizations find it difficult to match Kubernetes release
+cadence - only 3 releases per year will relax this situation.
+
+#### Distributors and downstream projects
+
+Downstream projects assemble their solution from different projects. Having
+fewer upgrades helps them to reduce complexity. For example, cloud providers
+will gain more room for upgrading their infrastructure.
+
+#### Contributors
+
+With a lower release cadence, contributors will gain more time for project
+enhancements, feature development, planning, and testing. It will provide more
+room for maintaining their mental health, prepare for events like KubeCon or
+work on the downstream integration.
+
+Through this proposal SIG Release's aim is to give contributors more
+flexibility to decide how to invest their time. It is explicitly _not_ to push
+contributors in doing more.
+
+#### SIG Release members
+
+By applying a cadence of 3 releases per year, SIG Release members will gain a
+reduced management overhead. There are also only 3 patch releases to maintain,
+which right now can overlap up to 4. SIG Release will gain more time to ensure
+a seamless transition from the previous release team to the next one. It is
+also possible to include more shadows if the role leads conclude that this is
+appropriate.
+
+### Risks and Mitigations
+
+#### Concentrating risk
+
+In theory a reduced release cadence will cause more changes for every release.
+This means that there will be an increased risk, which would usually be split
+up into 4 dedicated milestones rather than 3.
+
+SIG Release cannot mitigate this risk directly, but is able to track and
+influence it during each release cycle. It's the responsibility of SIG Release,
+together with SIG Testing and SIG Architecture, to identify new gaps and issues
+in the release cadence and mitigate them on a case-by-case basis.
+
+#### Attention to tests
+
+This KEP does not propose any change to the release cycle itself and assumes
+that the same periods for Code and Test Freeze. Assuming that, there is an
+increased risk for flakes and test failures. It will be the responsibility of
+SIG Release to mitigate this, together with the CI signal role. If we speak
```

Comment:
FTR - we're looking at it in WG Reliability group too - we're setting up a some tooling as we speak and are planning to propose https://docs.google.com/document/d/1P0GICwewHVYrlgsc_XJlLA3h_fOKub3sGkuogSPIliU/edit more officially soon.

@deads2k @stevekuznetsov

Reply:
Thanks for that input @wojtek-t!